RecoTracker/TkNavigation/plugins/CosmicNavigationSchool.cc

```diff
@@ -44,6 +44,7 @@ class dso_hidden CosmicNavigationSchool : public SimpleNavigationSchool {
   //FakeDetLayer* theFakeDetLayer;
   void linkBarrelLayers( SymmetricLayerFinder& symFinder) override;
   //void linkForwardLayers( SymmetricLayerFinder& symFinder); 
+  void establishInverseRelations() override {};
```

Comment:
Please clarify why this change is applied.
If the intention is to disallow calls to this method from the base pointer, then better add an assert or an exception.
Otherwise, the original 
```using SimpleNavigationSchool::establishInverseRelations;``` seems to make more sense.

A further improvement can be to replace the implementation of ```void establishInverseRelations( SymmetricLayerFinder& symFinder )``` with 
```c++
void establishInverseRelations( SymmetricLayerFinder& symFinder ){
  SimpleNavigationSchool::establishInverseRelations();
  buildAdditionalForwardLinks(symFinder); 
}
```

Reply:
This is another case where the derived classes do not see the function declarations for the base class SimpleNavigationSchool.